assets/js/src/newsletters/send.jsx

```diff
@@ -34,11 +34,13 @@ const NewsletterSend = createReactClass({ // eslint-disable-line react/prefer-es
       item: {},
       loading: true,
       thumbnailPromise: null,
+      authorizedEmailAddresses: [],
     };
   },
 
   componentDidMount: function componentDidMount() {
     this.loadItem(this.props.match.params.id);
+    this.loadAuthorizedEmailAddresses();
```

Comment:
It makes sense to do this API call only if sending method is set to MailPoet so I think it would be better to call it only for MailPoet sending method to save unnecessary API call.

Also this is not chained or combined with loadItem, which turns off loading state once it is done. There is a possibility that user will be able to fill the form but the authorized email validation will fail because emails will not be loaded. The probability is rather low but it the api call does another api call in background (Bridge API) so it is still possible.

MailPoet.Ajax.Post returns jQuery deferred objects which could be combined using [jQuery.when](https://api.jquery.com/jquery.when/)